openshift/e2e-verify-manifests.sh

```
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+oc get -f openshift/capi-operator-manifests/default/manifests.yaml
+
+# TODO: Update the array below when https://redhat.atlassian.net/browse/OCPCLOUD-3537 is done
+declare -a arr=("pod.not-exist.io" "svc.not-exist.io")
+
+for crd in "${arr[@]}"; do
+  echo "Checking if ${crd} exists on the cluster"
+  if oc get crd ${crd} ; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)